corehq/sql_db/routers.py

```diff
@@ -11,6 +11,7 @@
 ICDS_MODEL = 'icds_model'
 SCHEDULING_PARTITIONED_APP = 'scheduling_partitioned'
 WAREHOUSE_APP = 'warehouse'
+SYNCLOGS_APP = 'phone'
```

Comment:
@sravfeyn @snopoke I think this broke prod since this routes all `phone` app DB calls into the new synclogs db.

Reply:
yes, that's correct and expected. How did this break?